File: app/src/main/java/com/grace/placessearch/data/model/SuggestedVenuesResponse.java
package com.grace.placessearch.data.model;

import com.google.gson.annotations.Expose;
import com.google.gson.annotations.SerializedName;

public class SuggestedVenuesResponse {

    @SerializedName("meta")
    @Expose
    private Meta meta;
    @SerializedName("response")
    @Expose
    private SuggestedResponse suggestedResponse;

    public Meta getMeta() {
        return meta;
    }

    public void setMeta(Meta meta) {
        this.meta = meta;
    }

    public SuggestedResponse getResponse() {
        return suggestedResponse;
    }

    public void setResponse(SuggestedResponse suggestedResponse) {
        this.suggestedResponse = suggestedResponse;
    }

}

File: app/src/main/kotlin/com/grace/placessearch/common/data/model/SuggestedVenuesResponse.kt
package com.grace.placessearch.common.data.model

import com.google.gson.annotations.Expose
import com.google.gson.annotations.SerializedName

class SuggestedVenuesResponse {

    @SerializedName("meta")
    @Expose
    var meta: Meta? = null
    @SerializedName("response")
    @Expose
    var response: SuggestedResponse? = null

}